com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -50,6 +50,11 @@ internal struct AgentInfo
         /// </summary>
         public int episodeId;
 
+        /// <summary>
+        /// Team Manager identifier.
+        /// </summary>
+        public string teamManagerId;
```

Comment:
This shouldn't be public. I think a better approrach is to add an accessor for m_TeamManager, and add a method to the interface for an ID.